src/main/webapp/app/grading-system/grading-system.component.html

```diff
@@ -57,21 +79,45 @@ <h2 style="margin-bottom: 0.5em">
                 </tr>
             </thead>
             <tbody>
-                <tr *ngFor="let gradeStep of gradingScale.gradeSteps; let i = index">
+                <tr *ngFor="let gradeStep of gradingScale.gradeSteps; let i = index" class="grading-scale-table-row">
+                    <td>
+                        <input [(ngModel)]="gradeStep.lowerBoundPercentage" type="number" step="0.01" min="0" max="100" required (change)="setPoints(gradeStep, true)" />
+                    </td>
+                    <td>
+                        <input
+                            [(ngModel)]="gradeStep.lowerBoundPoints"
+                            type="number"
+                            step="0.01"
+                            min="0"
+                            [max]="getMaxPoints() || 999"
+                            [required]="maxPointsValid()"
```

Comment:
you have a lot of method calls here which can make the whole page very slow due to Angular's automatic change tracking. I suggest to reduce the number of method calls or to change the change tracking strategy (see e.g. the quiz edit page)